math/convertall/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
```

Comment:
Nice looking job for your first Portfile! Thanks for taking the leap.

Reply:
Thanks - I hope to slowly add more Portfiles, as I stumble across tools I use that aren't in MacPorts, or see one that has no maintainer